trimmerDecorator.rb

```diff
@@ -4,4 +4,4 @@ class TrimmerDecorator < Decorator
   def correct_name
     @nameable.correct_name.slice(0, 9)
```

Comment:
- [ ] Great job so far 👍 The `TrimmerDecorator` should return a maximum of 10 characters but yours returns a maximum of 9 characters. Kindly correct this 😄